generate-genesis.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+# Genesis Generator for Lean Quickstart
+# Generates validators.yaml, nodes.yaml, and .key files from validator-config.yaml and config.yaml
+# This tool eliminates the need for hardcoded genesis files
+
+set -e
+
+# ========================================
+# Usage and Help
+# ========================================
+show_usage() {
+    cat << EOF
+Usage: $0 <genesis-directory>
+
+Generate genesis configuration files (validators.yaml, nodes.yaml, and .key files)
```

Comment:
we have to use PK's genesis tool here and not generate our own tooling (so its docker image)